sdk/formrecognizer/Azure.AI.FormRecognizer/src/CopyModelOperation.cs

```diff
@@ -173,9 +173,9 @@ protected CopyModelOperation()
 
         async ValueTask<OperationState<DocumentModelDetails>> IOperation<DocumentModelDetails>.UpdateStateAsync(bool async, CancellationToken cancellationToken)
         {
-            Response<DocumentModelOperationDetails> response = async
-                    ? await _serviceClient.GetOperationAsync(Id, cancellationToken).ConfigureAwait(false)
-                    : _serviceClient.GetOperation(Id, cancellationToken);
+            Response<GetOperationResponse> response = async
+                ? await _serviceClient.GetOperationAsync(Id, cancellationToken).ConfigureAwait(false)
+                : _serviceClient.GetOperation(Id, cancellationToken);
```

Comment:
Originally we were just renaming the swagger-defined `GetOperationResponse` to `DocumentModelOperationDetails` and exposing it as part of public API.

Since we want to remove the inheritance between it and its base class, we need to create a whole new `DocumentModelOperationDetails` class and hide the original `GetOperationResponse` from the public API.